data/strings/strings.txt

```diff
@@ -5661,6 +5661,14 @@
     zh-Hans = 更新地图
     zh-Hant = 更新地圖
 
+  [google_play_services]
+    comment = Preference title
+    tags = android
+    en = Google Play Services
```

Comment:
@MetehanOzyurek what will be the Turkish string please?

Reply:
Ok 👌

Reply:
> tr = Google Play Hizmetleri

@pastk